game/src/core/systems/VelocitySystem.java

```diff
@@ -42,9 +46,11 @@
  */
 public final class VelocitySystem extends System {
 
+    protected boolean isHeroInFight;
```

Comment:
Ich mach das mal hier als zusammenfassung:
Ich verstehe die Intention hinter der Abfrage ob der Held kämpft oder nicht, da dies ja auch andere Animationen bedeutet.
Das hat in diesem System aber nichts zu suchen. Das Velocity-System ist nur für Animationen related zur Bewegung zuständig. 
Aktuell haben wir kein System für das Kämpfen (@Lena241 mach dafür mal ein Issue auf)

Reply:
Ok, dann nehme ich den Kampfmodus raus, oder soll ich diesen einfach nur auskommentieren?